php/libraries/NDB_BVL_Feedback.class.inc

```diff
@@ -469,7 +469,7 @@ class NDB_BVL_Feedback
         }
         $query .= " WHERE ft.Active ='Y'";
         if (!$hasReadPermission===true) {
-            $query            .= " AND FIND_IN_SET (c.CenterID,:CentID)";
+            $query            .= " AND FIND_IN_SET (s.CenterID,:CentID)";
```

Comment:
no space between SET and opening parenthesis. space between comma and colon

Reply:
done